css/index.css

```diff
@@ -1,3 +1,292 @@
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+ol, ul {
+	list-style: circle;
+}
+
+.main-content {
+    position: absolute;
+    margin: auto;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+	max-width: 900px;
+}
+
+header {
+    display: flex;
+    width: 100%;
+    margin: 20px 0;
+}
+
+nav {
+	display: flex;
+	justify-content: flex-start;
+    width: 100%;
+    
+}
+
+nav a {
+    margin: 0 80px 0 0;
+	margin-top: 1%;
+	text-decoration: none;
+	color: grey;
+}
+
+
+.top-feature {
+	display: flex;
+	justify-content: space-evenly;
+	margin: 50px 0 ;
+}
+
+.feature-text {
+	width: 30%;
+	height: 220px;
+	margin: 5% 10%;
+}
+
+.feature-text h2 {
+	font-size: 70px;
+	display: flex;
+	justify-content: center;
+}
+
+.get-started-btn {
+    margin-left: 30px;
+	width: 200px;
+	height: 35px;
+	background-color: #fff;
+}
+
+.top-content {
+	display: flex;
+	margin: 4% 0;
+	justify-content: space-around;
+}
+
+.features {
+	width: 45%;
+}
+
+.about {
+	width: 45%;
+}
+
+.middle-image {
+	display: flex;
+	justify-content: center;
+}
+
+.bottom-content {
+	display: flex;
+	justify-content: space-around;
+	margin: 4% 0;
+}
+
+.services {
+	width: 30%;
+}
+
+.product {
+	width: 30%;
+}
+
+.vision {
+	width: 30%;
+}
+
+.main-address {
+	display: flex;
+	flex-direction: column;
+}
+
+.main-phone {
+	margin: 2% 0;
+}
+
+.copyright {
+	display: flex;
+	justify-content: center;
+}
+
+/* Tablet View */
+
+@media (min-width: 768px) and (max-width: 1024px) {
+  
+    .feature-pic {
+        display: none;
+    }
+    
+    header {
+        display: flex;
+        flex-wrap: wrap-reverse;
+    }
+
+    .logo {
+       margin-left: 360px; 
+    }
+
+    .nav {
+        display: flex;
+        justify-content: space-between;
+        width: 100%;
+    }
+
+    .nav a {
+        margin: 20px 0;
+    }
+
+    .top-feature {
+        margin: 10px 0 80px;
+    }
+    
+}
+
+/* Phone View */
+
+  @media (max-width: 768px) {
+
+
+    header {
+        display: flex;
+        flex-wrap: wrap-reverse;
+    }
+
+    .logo {
+        margin: 0 0 20px 110px; 
+     }
+
+    .feature-pic {
+        display: none;
+    }
+
+    .nav {
+        display: flex;
+        flex-direction: column;
+        align-content: center;
+
+    }
+    .nav a {
+        padding-top: 3%;
+        margin: 0px;
+        width: 100%;
+        height: 45px;
```

Comment:
The text within the anchor tags don't appeared to be centered. Try removing the fixed height and have equal padding for both top and bottom. How does that work for you?